src/tools/dartpad/index.md

```diff
@@ -45,7 +45,7 @@ try running some samples and then creating a simple command-line app.
   Go to <a href="{{site.dartpad}}" target="_blank" rel="noopener">DartPad.</a>
 
   A sample appears on the left and the output appears on the right.
```

Comment:
I think auto-run if off for Dart samples, to avoid problems with endless loops. Also, if the reader has played with DartPad before, they might not see a sample. Maybe:

```suggestion
  Dart code appears on the left, and
  a place for the output appears on the right.
```